### PR TITLE
Update to latest releases and specs

### PR DESCRIPTION
**Pull request by pault84**
Signed-off-by: Paul <paul@portworx.com>

**What this PR does / why we need it**:
Moves the helm charts to the latest versions.
Adds monitoring if desired.

We're currently a bit behind on versions and we didn't have monitoring present yet.
Rancher and IKS are even further behind.

### CODE REVIEW

File: charts/portworx/templates/portworx-monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+  ports:
+    - name: web
+      port: 9093
+      protocol: TCP
+      targetPort: 9093
+  selector:
+    alertmanager: portworx
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
```
Comment (harsh-px) [inline charts/portworx/templates/portworx-monitor.yaml]: wouldn't this cause the helm install to fail since it won't find the kind?

Comment (pault84) [reply charts/portworx/templates/portworx-monitor.yaml]: Pre-requisites in doc i'll add them there.

File: charts/portworx/values.yaml [hunk 2]
```diff
@@ -26,9 +26,10 @@ customRegistryURL:
 registrySecret:
 
 lighthouse: true
-lighthouseVersion: 2.0.1
+lighthouseVersion: 2.0.2
 lighthouseSyncVersion: 0.3
 lighthouseStorkConnectorVersion: 0.1
+pxCentral: false
```
Comment (harsh-px) [inline charts/portworx/values.yaml]: let's use the same param names as install.portworx.com. So let's use `monitoring`.

File: charts/portworx/templates/portworx-monitor.yaml (hunk 1 above)
Comment (pault84) [reply charts/portworx/templates/portworx-monitor.yaml]: Or should i go straight to the newest way with lighthouse deploying the CRD's ?

**Pull request merged**